apps/meteor/app/livechat/server/startup.ts

```diff
@@ -18,7 +18,8 @@ import { RoutingManager } from './lib/RoutingManager';
 import './roomAccessValidator.internalService';
 
 Meteor.startup(async () => {
-	roomCoordinator.setRoomFind('l', (_id) => LivechatRooms.findOneById(_id));
+	// TODO: remove promise.await when roomCoordinator supports async functions
```

Comment:
roomCoordinator supports async now =)

Reply:
U sure? The file doesn't have any async thing in it yet 🤔